src/sentry/api/authentication.py

```diff
@@ -189,6 +190,9 @@ class TokenAuthentication(StandardAuthentication):
     token_name = b"bearer"
 
     def authenticate_credentials(self, request: Request, token_str):
+        if token_str.startswith(SENTRY_JWT_PREFIX):
```

Comment:
@mydea hmm why do we need this?

Reply:
This is because some code basically goes through a list of given authentication methods in order, based on the `token_name`. however this is the same for the user api token & org token (both are bearer tokens).  So it will first try to check it against the `TokenAuthentication`, where it returns `None` if we detect it is an org token. Then it will check the OrgTokenAuthentication, where it should match and actually check it. So this is basically an extension of this in the base `StandardAuthentication` class:

```py
def authenticate(self, request: Request):
        auth = get_authorization_header(request).split()

        if not auth or auth[0].lower() != self.token_name:
            return None
```